src/daemon/engine/CMakeLists.txt

```diff
@@ -36,18 +36,19 @@ target_include_directories(udpard
         INTERFACE SYSTEM ${submodules_dir}/libudpard/libudpard
 )
 
-add_library(canard
-        ${submodules_dir}/libcanard/libcanard/canard.c
-)
-target_include_directories(canard
-        INTERFACE SYSTEM ${submodules_dir}/libcanard/libcanard
-)
+if (${PLATFORM_OS_TYPE} STREQUAL "linux")
+    add_library(canard
+            ${submodules_dir}/libcanard/libcanard/canard.c
+    )
+    target_include_directories(canard
+            INTERFACE SYSTEM ${submodules_dir}/libcanard/libcanard
+    )
+endif ()
```

Comment:
This seems to imply that we cannot use CAN in macOS at all, but that's only true because we only support SocketCAN. One could easily add a driver for SLCAN or gs-usb and use CAN on any OS in this manner, so this exclusion is misleading here.